lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -129,8 +129,10 @@ def correction(node)
         def frozen_strings?
```

Comment:
This function says strings are frozen in preparation of `Style/FrozenStringLiteralComment` doing its autocorrect. Introduced in #9773. I don't love this since it now gets complicated when the default changes. The change to the function works but I feel pretty meh about it.

Why no `autocorrect_incompatible_with`? `EmptyLiteral` comes before `FrozenStringLiteralComment` alphabetically. I played around with letting cops say their autocorrect is more important than others but didn't like how that turned out.

I would rather get rid of this workaround entirely. The user runs unsafe autocorrect and should not be surprised when things break.

Reply:
No argument from me.